config.properties.template

```diff
@@ -175,12 +182,19 @@ obligatory_transfer=DODUO,RATTATA,CATERPIE,PIDGEY
 # list of pokemon you always want to evolve before transfer to maximize XP
 evolve_before_transfer=CATERPIE,RATTATA,WEEDLE,PIDGEY
 
+# list of pokemon you NEVER want to catch, regardless of cp/iv
+never_catch_pokemon=
+
 # The stack of evolves needed to pop lucky egg and evolve all
 evolve_stack_limit=100
 
 # Use lucky egg before evolves (1 = yes, anything else = no)
 use_lucky_egg=1
 
+# Should the bot wait between each evolution and how long? -1 to disable, time to wait in milliseconds (it will be randomized alittle around the given value)
```

Comment:
@AlexBoulyga doing like the app is what we want :P ... but choice is of the user